cluster/cluster_service.proto

```diff
@@ -40,6 +40,8 @@ service TypeDBCluster {
 
     // User API
     rpc user_password (ClusterUser.Password.Req) returns (ClusterUser.Password.Res);
+    rpc user_password_admin (ClusterUser.PasswordAdmin.Req) returns (ClusterUser.PasswordAdmin.Res);
+    rpc user_expiry_days (ClusterUser.ExpiryDays.Req) returns (ClusterUser.ExpiryDays.Res);
```

Comment:
let's use:
`user_password_set` - the admin can set another user's password
`user_password_update` - a user can update new users


The final set of APIs will be:
```
    // User Manager API -- these are mostly available to only admins
    rpc users_contains (ClusterUserManager.Contains.Req) returns (ClusterUserManager.Contains.Res);
    rpc users_create (ClusterUserManager.Create.Req) returns (ClusterUserManager.Create.Res);
** rpc users_delete (ClusterUserManager.Delete.Req) returns (ClusterUserManager.Delete.Res);
    rpc users_all (ClusterUserManager.All.Req) returns (ClusterUserManager.All.Res); 
** rpc users_password_set (ClusterUserManager.PasswordSet.Req) returns (ClusterUserManager.PasswordSet.Res);
** rpc users_get .... this one will be available to each user OR admins


    // User API -- these are accessible to the logged in user
** rpc user_password_update(ClusterUser.PasswordUpdate.Req) returns (ClusterUser.PasswordUpdate.Res);
    rpc user_token (ClusterUser.Token.Req) returns (ClusterUser.Token.Res);
```

Reply:
The `User` object that is returned from `_get` and `_all` contains the fields:
```
name: string
passwordExpiryDays: int
```